src/system/process.py

```diff
@@ -66,9 +66,9 @@ def terminate(self) -> int:
                 try:
                     for item in proc.open_files():
                         if self.stdout.name == item.path:
-                            raise Exception(f"stdout {item} is being used by process {proc}")
-                except Exception as err:
-                    logging.error(f"{err.args}")
+                            logging.error(f"stdout {item} is being used by process {proc}")
+                except Exception:
+                    pass
```

Comment:
Catching and exception and not doing anything is not making sense. Can you elaborate what you are trying to do here?

Reply:
These exceptions caught are mainly some process that were not related to our testing or they were already closed during the iteration so we may not have permission to go through it. I have seen quite few examples of using this regarding of whether certain file is closed/not used by any process. e.g. https://stackoverflow.com/questions/64599502/how-can-i-check-if-a-file-is-actively-open-in-another-process-outside-of-my-pyth. I think it's safe to do this.